lib/modem_info/modem_info_json.c

```diff
@@ -277,6 +286,9 @@ int modem_info_json_string_encode(struct modem_param_info *modem,
 
 		total_len += ret;
 		json_add_obj(root_obj, MODEM_INFO_JSON_KEY_DEV_INF, device_obj);
+
+		/* device_obj is not part of the root_obj */
```

Comment:
```suggestion
		/* device_obj is now part of the root_obj */
```

Reply:
Thanks, I've fixed all of the reported typos.